gitignore.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
One of the goals of this project is to set up linters for [HTML and CSS](https://github.com/microverseinc/linters-config) correctly in this repository.

Kindly consider adding and installing linters in your project and ensure that the linter results are green, this means that the last commit should have a green check mark (✅) or a summary of the passed checks is in the PR.

[Reminder]:
Don't to forget to add ` /node_modules` to your `.gitignore` file as well 👍🏽